app/presenters/publishing_api_presenters/detailed_guide.rb

```diff
@@ -83,4 +84,24 @@ def unpublishing_explanation
       Whitehall::GovspeakRenderer.new.govspeak_to_html(item.unpublishing.explanation)
     end
   end
+
+  def applicable_nations
+    all_nations = %w(England Scotland Wales Northern\ Ireland)
+    inapplicabilities = item.nation_inapplicabilities
+    applies_to_all_nations = inapplicabilities.empty?
+
+    if applies_to_all_nations
+      nations = []
+    else
+      excluded_nations = inapplicabilities
+        .map(&:nation)
+        .map(&:name)
```

Comment:
I think we need to translate these based on the locale of the requested Detailed Guide, but it doesn't look like we have translations for them.